### ISSUE TITLE
syncdb not working

### ISSUE THREAD

**Issue by Heldroe** [ogirardot/django-achievements#2]
Hi,

When I do a syncdb, my achievements are not collected. But if I comment lines 21 and 22 of achievements/models.py, they are collected and stored in the database.
I am not using south and don't really understand what is going on.

Are these lines important ? Is this not a bug ?

Thanks

**Comment by ogirardot** [comment]
it may be :) 
i'll try to reproduce it right now, but in the meantime, 
the achievements app is in your INSTALLED_APP setting variable right ?

**Comment by ogirardot** [comment]
Ok so this is a bug, 
when the code you commented is called with south the kwargs variable is :

```
{'app': 'achievements', 
'sender': None, 
 'signal': <django.dispatch.dispatcher.Signal object at 0x10537f610>}
```

and when it is called from syncdb (pure and simple) i get :

```
{'sender': <module 'achievements.models' from '.../achievements/models.py'>, 
 'app': <module 'achievements.models' from '.../achievements/models.py'>, 
 'db': 'default', 
 'created_models': set([]), 
 'verbosity': 1, 
 'signal': <django.dispatch.dispatcher.Signal object at 0x101b2ffd0>,
 'interactive': True}
```

I'll fix it right away.

**Comment by ogirardot** [comment]
hi there, i published 0.0.6 just now, it should fix your problems, 
can you please test it when you have the time ? 
thx

**Comment by Heldroe** [comment]
I just tested it and confirm it now works.
Thanks :)

**Comment by ogirardot** [comment]
thx, did you have any issues or comments on how to use it ?

**Comment by Heldroe** [comment]
It is pretty easy to use and works out of the box.
One thing was the appconf dependency that is not documented but it is reported in issue #3.

Maybe I just miss a way to get one user's achievements directly (the same way as we get the score), same thing for the whole list of achievements.

**Comment by ogirardot** [comment]
you mean template tags ? or util methods ?

2011/11/30 Heldroe <
reply@reply.github.com

**Comment by Heldroe** [comment]
Util methods. Template tags may be too much.

**Comment by ogirardot** [comment]
ok working on it.
in the meantime i fixed the dependency problem in 0.0.7.
Regards.
